bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tdirectory.py

```diff
@@ -0,0 +1,23 @@
+# Author: Danilo Piparo, Stefan Wunsch CERN  08/2018
+
+################################################################################
+# Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from libROOTPython import PythonizeTDirectory
+from ROOT import instant_pythonization
+import cppyy
+
+# This is an instant pythonization. As such, no argument is needed since we know
+# what we are pythonizing.
+
+@instant_pythonization
```

Comment:
I know this is just "internal programming model", but what about reusing the @pythonization decorator with a parameter that specifies that the pythonization is greedy (default would be lazy)

Reply:
I like your idea though of also decorating the greedy pythonizations instead of just calling them directly from ROOT.py when importing ROOT, it is elegant.

Reply:
"mine" --> @stwunsch :)
That can be done easily, I just would like to converge on the name (greedy or instant) before.

Reply:
See my answer above for the greedy/instant discussion.

Reply:
Then in the documentation we just say that if the `lazy` parameter is set to `False`, the pythonizations will be done as soon as one imports ROOT.